prototype warnings in paqa_devs.c

Build PortAudio on a Mac using:  ./configure && make.

EXPECT no errors or warnings

ACTUAL
```
./qa/paqa_devs.c:518:25: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
static void RunQuickTest()
                        ^
                         void
```

--- a/qa/paqa_devs.c
+++ b/qa/paqa_devs.c
@@ -140,6 +140,7 @@ static int QaCallback( const void *inputBuffer, void *outputBuffer,
                        const PaStreamCallbackTimeInfo* timeInfo,
                        PaStreamCallbackFlags statusFlags,
                        void *userData );
+static void RunQuickTest( void );
 
 static void PaQaSetupData(PaQaData *myData,
                           const PaQaTestParameters *parameters)
@@ -515,7 +516,7 @@ error:
 }
 
 
-static void RunQuickTest()
+static void RunQuickTest( void )
 {
     PaQaTestParameters parameters = kDefaultTestParameters;
 

